templates/Coaster/show.html.twig

```diff
@@ -335,7 +333,36 @@
           });
       };
 
+      var loadReviews = function(page) {
+        $.ajax({
+              url: Routing.generate('coaster_reviews_ajax_load', {
+                  'slug': '{{ coaster.slug }}',
+                  'page': page || 1,
+                  '_locale': '{{ app.request.locale }}',
+              }),
+              type: 'GET',
+          }).done(function (data) {
+              $('#coaster-reviews').replaceWith(data);
+              if(page != undefined) {
+                $([document.documentElement, document.body]).animate({
+                  scrollTop: $("#coaster-photos").offset().top
+              }, 1000);
+              }
+              var buttons = document.getElementsByClassName('page-link');
+              Array.prototype.forEach.call(buttons, function(el) {
+                  if(el.nodeName == "A") {
```

Comment:
take a look at the ranking twig, there is a way to use jQuery to select all "a" elements